include/debug/stack.h

```diff
@@ -15,6 +15,12 @@
 #include <logging/log.h>
 #include <stdbool.h>
 
+extern K_THREAD_STACK_DEFINE(z_main_stack, CONFIG_MAIN_STACK_SIZE);
```

Comment:
do not put these here, they are private to the kernel

Reply:
I need an alternative so that I can debug the main stack.

Reply:
Removed